server/application/application.go

```diff
@@ -2028,16 +2053,16 @@ func (s *Server) Sync(ctx context.Context, syncReq *application.ApplicationSyncR
 	if len(syncReq.Resources) > 0 {
 		partial = "partial "
 	}
-	var reason string
+	var action string
```

Comment:
this is an "action", not a "reason".

argo.EventReasonOperationStarted is the reason, so makes sense to update this.